main/EventSystem.cpp

```diff
@@ -1235,7 +1254,28 @@ void CEventSystem::EvaluateEvent(const std::string &reason, const uint64_t Devic
 		{
 			if (reason == "device" && filename.find("_device_") != std::string::npos)
 			{
-				EvaluateLua(reason, lua_Dir + filename, "", DeviceID, devname, nValue, sValue, nValueWording, 0);
+				bDeviceFileFound = false;
```

Comment:
Thanks for the feedback - yes, I went through my scripts and renamed the ones impacted.
Good for the wiki !